src/tr1/game/output.c

```diff
@@ -93,6 +94,64 @@ static void M_CalcRoomVertices(const ROOM_MESH *mesh);
 static void M_CalcRoomVerticesWibble(const ROOM_MESH *mesh);
 static void M_CalcWibbleTable(void);
 
+static void M_DrawSphere(const XYZ_32 pos, const int32_t radius)
```

Comment:
I wasn't sure what to do here about `XYZ_32` because `M_CalcVertex` takes `XYZ_16`. There's some type conversions throughout the codebase.